ui-frontend/packages/catalog-ui-search/src/main/webapp/behaviors/selection.behavior.js

```diff
@@ -60,8 +62,17 @@ Behaviors.addBehavior(
       this.interpretClick(event)
       this.updateStateOnClick(event)
     },
+    handleCheckboxClick(event) {
+      this.isCheckboxClick = true
+    },
     interpretClick(event) {
-      if (event.altKey || this.isTextSelection() || this.isDoubleClick(event)) {
+      if (
+        event.altKey ||
+        this.isTextSelection() ||
+        this.isDoubleClick(event) ||
+        this.isCheckboxClick
+      ) {
+        this.isCheckboxClick = false
```

Comment:
❓ not sure if we can word this better, or maybe im not understanding the logic - are we toggling `isCheckboxClick`? If a user clicks a checkbox and then immediately clicks another checkbox, would `isCheckboxClick` be false?